docs/setup.md

```diff
@@ -33,17 +33,15 @@ Combined with the changes above, this might look like:
 
 ### Monorepos with independent versions
 
-For a monorepo with independent versions, you will want to:
+For a monorepo with independent versions, you will want to add a `release.config.json` with a `versioningStrategy` of `"independent"`.
```

Comment:
Note that we don't actually use the `release.config.json` right now. In fact this documentation is mostly aspirational and reflects future plans. Perhaps I ought to make that clear in another set of changes, but I wanted to call that out in case it was confusing.

Reply:
This file is used by https://github.com/MetaMask/action-publish-release

Reply:
Oh, right, we did decide to make of it there, didn't we. Okay, I'll add some documentation to that action to mention that.